CMakeLists.txt

```diff
@@ -532,7 +564,8 @@ if(WIN32)
   move_dll(${LOVR_GLFW})
   move_dll(${LOVR_LUA})
   move_dll(${LOVR_ODE})
-  move_dll(${LOVR_OPENAL})
-  move_dll(${LOVR_OPENVR})
+  if (LOVR_USE_OPENVR)
+    move_dll(${LOVR_OPENVR})
+  endif()
```

Comment:
This is removing the OpenAL .dll on windows builds?  The `if` shouldn't be necessary either since `move_dll` checks to see if the target exists before doing the copy.

Reply:
Wait, now I see the problem. I will remove this change entirely.